.github/workflows/integrationtests.yaml

```diff
@@ -112,3 +112,21 @@ jobs:
 
       - name: Daps Integration Tests
         run: ./gradlew -p extensions/iam/daps test -DincludeTags="DapsIntegrationTest"
+
+  OpenTelemetry-Integration-Tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'temurin'
+          java-version: '11'
```

Comment:
17? (align with upstream)

Reply:
Fixed it in another PR and wait for the merge to resolve the comment.